rust/Cargo.lock

```diff
@@ -484,22 +491,14 @@ dependencies = [
  "bitflags",
 ]
 
-[[package]]
-name = "cloudabi"
-version = "0.1.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "4344512281c643ae7638bbabc3af17a11307803ec8f0fcad9fae512a8bf36467"
-dependencies = [
- "bitflags",
-]
-
 [[package]]
 name = "combine"
-version = "4.4.0"
+version = "4.5.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "b9417a0c314565e2abffaece67e95a8cb51f9238cd39f3764d9dfdf09e72b20c"
+checksum = "01199925a18b00193570e3d70cfe57dcb647eb167c29851983e76fc39e2fee39"
 dependencies = [
- "bytes",
+ "bytes 0.5.6",
```

Comment:
I'm currently testing the changes on k8s and get some really weird redis errors (NoSeedDict, LengthMisMatch). I wonder if it comes from the fact that we have two versions of `bytes` in `combine` which is used by `redis`

Reply:
we should definitely rule out version conflicts first.

Reply:
I think it is the same issue https://github.com/mitsuhiko/redis-rs/issues/429
although in our case it compiles and only fails at runtime, which is scary

Reply:
Hm I might have been wrong with `bytes`. It seems that there is an issue in k8s that only occurs once we create (not restart) a coordinator pod. The bug itself is really strange. redis does return `None` when fetching the dictionaries after a sum or update phase, although the data exists in redis. That error has never happened before. I need to take a look on this later. The error also occurs with the code from master so I think it is safe to merge your pr. 

I think we can also revert the last commit if you like.

Reply:
Ok the errors i got are actually related to k8s. When re-creating a pod, it can happen that two coordinators exist at the same time and both then access redis

Reply:
mhh ok, strange error, but good catch! i'll revert the last commit.